org.eclipse.wildwebdeveloper.xml/src/org/eclipse/wildwebdeveloper/xml/MavenArtifactClassPathProvider.java

```diff
@@ -0,0 +1,12 @@
+package org.eclipse.wildwebdeveloper.xml;
+
+import java.io.File;
+
+public class MavenArtifactClassPathProvider implements XMLLSClasspathExtensionProvider {
```

Comment:
This should be in the m2e patch that's currently on Gerrit.

Reply:
I kept it in the most recent commit but am going to remove it now and add it to the gerrit m2e patch.